modules/rest-api/src/main/java/org/apache/ignite/internal/rest/api/recovery/system/SystemDisasterRecoveryApi.java

```diff
@@ -51,6 +51,20 @@ public interface SystemDisasterRecoveryApi {
     @ApiResponse(responseCode = "400", description = "Bad request.",
             content = @Content(mediaType = MediaType.PROBLEM_JSON, schema = @Schema(implementation = Problem.class)))
     @Consumes(MediaType.APPLICATION_JSON)
-    @Produces(MediaType.PROBLEM_JSON)
+    @Produces({MediaType.APPLICATION_JSON, MediaType.PROBLEM_JSON})
     CompletableFuture<Void> reset(@Body ResetClusterRequest command);
+
+    @Post("migrate")
+    @Operation(
+            operationId = "migrate",
+            description = "Migrates nodes from old cluster to new (repaired) cluster."
+    )
+    @ApiResponse(responseCode = "200", description = "Migration initiated.")
+    @ApiResponse(responseCode = "500", description = "Internal error.",
```

Comment:
There are quite a lot of `description = "Internal error."` in the project, maybe we can turn it into a constant with a separate ticket, what do you think?

Reply:
Added a ticket https://issues.apache.org/jira/browse/IGNITE-23274 and TODOs